app/networkapi/settings.py

```diff
@@ -203,6 +202,15 @@
     'mezzanine.generic.models.ThreadedComment',
 ]
 
+ADMIN_MENU_ORDER = (
+    ('Content', ('pages.Page', 'blog.BlogPost', 'generic.ThreadedComment',
+                 ('Media Library', 'media-library'))),
+    ('Site', ('sites.Site', 'redirects.Redirect', 'conf.Setting')),
```

Comment:
It already behaves that way- I created a new user as a test and made them staff, but they couldn't see the settings. Lines 206-208 are simply the default value for this property.